app/views/guides/show.html.erb

```diff
@@ -2,20 +2,18 @@
 <% content_for(:meta_description, @guide.description) %>
 
 <% content_for :sidebar do %>
-  <% if @journey.include?(@guide) %>
-    <div class="link-promo ga-journey-sidebar">
-      <h2 class="link-promo__heading">6 steps you need to take</h2>
-
-      <p>
-        <a href="/6-steps-you-need-to-take">Read the 6 steps overview</a>
-      </p>
-
-      <hr class="hr-thin">
-
-      <% cache(@guide.slug) do %>
-        <%= render 'components/journey_nav', journey: @journey, current_step: @guide %>
-      <% end %>
-    </div>
+  <% if journey_navigation.active? %>
+    <% cache(@guide.slug) do %>
```

Comment:
the journey nav being active will also be determined based on the slug, so maybe we can move the caching one line up?